index.html

```diff
@@ -4,10 +4,176 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-    <link rel="stylesheet" href="mobile-version-style.css">
+    <title>My Portfolio Mobile Version</title>
+    <link rel="shortcut icon" href="Images/favicon.ico"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;500;600;700&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    
+    <main id="card-container">
```

Comment:
Hello @HFG43, I know you worked hard to finish this milestone 😃. As I can see in your project, you missed some sections. Ensure you added all sections such as `header`, and `home page`. You may check the Figma [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=t6rOGNfHIv0gVnpn-0) 👍 

![toolbar-mobile](https://user-images.githubusercontent.com/57408419/221090433-bb0682b9-498f-4213-b1bf-3fed94b28921.png)
![headline-mobile](https://user-images.githubusercontent.com/57408419/221090459-ae0897e2-1893-45d7-9fe4-a640d4cc58f7.png)
![works-mobile](https://user-images.githubusercontent.com/57408419/221090488-9f6c067c-7267-44d5-8100-dbc33f9c6c69.png)
<img width="188" alt="about-myself-mobile" src="https://user-images.githubusercontent.com/57408419/221090500-68f7b790-246c-4a83-afac-fab96df0c577.png">